### PR TITLE
Fix missing assets bug

### PR DESCRIPTION
**Pull request by svbergerem**
#5755 broke some cukes. Let's see if this PR can fix that.

### CODE REVIEW

File: app/views/profiles/_edit_public.haml [hunk 1]
```diff
@@ -1,10 +1,4 @@
--#   Copyright (c) 2010-2011, Diaspora Inc.  This file is
--#   licensed under the Affero General Public License version 3 or later.  See
--#   the COPYRIGHT file.
-
 - content_for :head do
-  = javascript_include_tag :profile
```
Comment (svbergerem) [inline app/views/profiles/_edit_public.haml]: `jquery.autoSuggest.custom` was before in that file which we already load in `main.js` and `mobile/mobile.js`.

File: app/views/status_messages/bookmarklet.html.haml [hunk 2]
```diff
@@ -1,9 +1,3 @@
--#   Copyright (c) 2010-2012, Diaspora Inc.  This file is
--#   licensed under the Affero General Public License version 3 or later.  See
--#   the COPYRIGHT file.
-
-= javascript_include_tag :home
```
Comment (svbergerem) [inline app/views/status_messages/bookmarklet.html.haml]: `fileuploader-custom` was before in that file which we already load in `main.js`.

**Comment by jhass** [comment]
Sorry for merging too early, and thanks again!

**Comment by svbergerem** [comment]
@jhass No problem. Travis is behaving really weird today.

**Pull request merged**